docs/resources/Channel.md

```diff
@@ -661,6 +661,10 @@ Get a list of users that reacted with this emoji. Returns an array of [user](#DO
 
 Deletes all reactions on a message. This endpoint requires the 'MANAGE_MESSAGES' permission to be present on the current user.
 
+## Delete All Reactions for Emoji % DELETE /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/messages/{message.id#DOCS_RESOURCES_CHANNEL/message-object}/reactions/{emoji.name#DOCS_RESOURCES_EMOJI/emoji-object}:{emoji.id#DOCS_RESOURCES_EMOJI/emoji-object}
```

Comment:
While `/{emoji.name#DOCS_RESOURCES_EMOJI/emoji-object}:{emoji.id#DOCS_RESOURCES_EMOJI/emoji-object}` isn't exactly wrong, it doesn't match up with how the other reaction endpoints annotate emoji; as `/{emoji#DOCS_RESOURCES_EMOJI/emoji-object}` (presumably relying on the comment on line 638). It would make sense to either change this to fit the format seen elsewhere or to change the documentation for the other reaction endpoints to fit this new style.

Reply:
Probably makes more sense to have it explicitly say `emoji.name`:`emoji.id` across the docs, for clarity. Otherwise you have to look through docs to figure out what `emoji` should be represented as.